app/markets/ActivityTable.tsx

```diff
@@ -6,81 +6,109 @@ import {
           <TableHead className="text-text-low-em">Users</TableHead>
           <TableHead className="text-text-low-em">Action</TableHead>
           <TableHead className="text-text-low-em">Shares</TableHead>
-          <TableHead className="text-text-low-em text-right">Date</TableHead>
+          <TableHead className="text-right text-text-low-em">Date</TableHead>
         </TableRow>
       </TableHeader>
-      <TableBody className="text-base font-semibold">
-        {activities.map(activity => (
-          <TableRow key={activity.user}>
-            <TableCell className="text-text-high-em">
-              {shortenAddress(activity.user)}
-            </TableCell>
-            <TableCell className="">
-              <Tag
-                size="xs"
-                colorScheme={activity.action === 'Yes' ? 'success' : 'danger'}
-                className="w-fit uppercase"
-              >
-                {activity.action}
-              </Tag>
-            </TableCell>
-            <TableCell className="text-text-high-em">{activity.shares}</TableCell>
-            <TableCell className="text-text-low-em text-right">
-              {formatDateTime(activity.date)}
-            </TableCell>
-          </TableRow>
-        ))}
-      </TableBody>
+      {isLoading ? (
+        <LoadingSkeleton />
+      ) : (
+        <TableBody className="text-base font-semibold">
+          {activities.map(activity => (
+            <TableRow key={activity.transactionHash}>
+              <TableCell className="text-text-high-em">
+                <a
+                  href={gnosisExplorerAddress(activity.user.id)}
+                  className="hover:underline"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {shortenAddress(activity.user.id)}
+                </a>
+              </TableCell>
+              <TableCell>
+                <Tag
+                  size="xs"
+                  colorScheme={getTagColorScheme(activity.transactionType)}
+                  className="w-fit uppercase"
+                >
+                  {activity.transactionType}
+                </Tag>
+              </TableCell>
+              <TableCell className="truncate text-text-high-em">
+                {formatEtherWithFixedDecimals(activity.sharesOrPoolTokenAmount)}
+              </TableCell>
+              <TableCell className="text-right text-text-low-em">
+                {formatDateTime(activity.creationTimestamp)}
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      )}
     </Table>
   );
 };
+
+const LoadingSkeleton = () => (
+  <TableBody className="text-base font-semibold">
+    {[1, 2, 3, 4, 5, 6, 7, 8].map(fakeActivity => (
+      <TableRow key={fakeActivity}>
+        <TableCell className="text-text-high-em">
+          <div className="h-[18px] w-[90px] animate-pulse rounded-8 bg-outline-low-em"></div>
+        </TableCell>
+        <TableCell className="">
```

Comment:
Do we need this className or it's a typo and we can remove it?

Reply:
typo